lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ContinuousRankedProbabilityScoreMinimisers.py

```diff
@@ -124,8 +124,7 @@ def setUp(self):
             np.float64)
 
 
-class Test_normal_crps_minimiser(
-        SetupGaussianInputs, EnsembleCalibrationAssertions):
+class Test_normal_crps_minimiser(SetupGaussianInputs):
```

Comment:
If the tests are no longer minimising should this be renamed?

Reply:
As I can see that the names of the `normal_crps_minimiser` functions are causing confusion, as these functions are not actually doing any minimising but rather they are the functions that will be minimised, I've renamed the functions and associated unit tests to avoid misleading developers.

Reply:
Sounds good, thanks @gavinevans